examples/allocation-endpoint/README.md

```diff
@@ -0,0 +1,104 @@
+# Allocation Endpoint
+
+This is a sample solution to enable an Allocation Endpoint proxy in front of GKE Agones clusters.
+
+In this folder you will find:
+
+1. Terraform modules that creates resources in GCP
+2. The Allocation Endpoint Proxy code that gets deployed to Cloud Run
+3. An ESP container that gets deployed to agones-allocator
```

Comment:
An ESP container that gets deployed as a sidecar alongside the agones-allocator

Reply:
Also, can we link "ESP" to some google cloud documentation describing what it is?